kiwi/boot/functions.sh

```diff
@@ -5525,18 +5540,25 @@ function waitForStorageDevice {
     # which could be a whole disk or a partition.
     # the function will wait until the size of the
     # storage device could be obtained or the check
-    # counter equals 4
+    # counter equals 30. For USB devices it waits
+    # only until check counter equals 2.
     # ----
     local IFS=$IFS_ORIG
     local device=$1
     local check=0
+    local transport="`getDeviceTransportType $device`"
+    if [[ $transport == *"usb"* ]];then
```

Comment:
@davidcassany @schaefi And you can also use a regexp:
```bash
if [[ $transport =~ "usb" ]]; then
...
fi
```

P.S. Please keep a whitespace after `;` and before `then` :wink: